app/models/katello/system.rb

```diff
@@ -182,6 +188,14 @@ def consumed_pool_ids=(attributes)
       end
     end
 
+    def update_foreman_facts
+      return unless self.foreman_host
+      rhsm_facts = self.facts
+      rhsm_facts[:_type] = RhsmFactName::FACT_TYPE
+      rhsm_facts[:_timestamp] = DateTime.now
```

Comment:
fact import actually expects a string (or something that responds to .to_datetime) so i've change it to use .to_s